.travis.yml

```diff
@@ -1,6 +1,22 @@
-language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4.2.4"
+language: c++
+env:
+  - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
+  - NODE_VERSION="0.12"
+  - NODE_VERSION="4.2.4"
```

Comment:
Two things:
1. I'd say just change this to `4`
2. Can you add `5`, and/or `stable`?